README.md

```diff
@@ -4,3 +4,46 @@ FullStack Matching Software for CIC
 Current considerations:
   1. Technical Algorithm to be used by backend -> simple keyword matching sufficent or a learning algorithm (database would need to be provided)
   2. Where will application be stored currently planning to be setup as a one button full stack java application but having it stored in server would be ideal
+
+How to Run the Project
+
+  Download the project
+    1. Navigate to https://github.com/Tsz-Fung-Ryan/BondsForSuccess in a web browser
+    2. Click the green `Code` button
+    3. Click the `Download ZIP` button
+    4. Unzip the downloaded zip file somewhere
+
+  Running the backend
+    1. Install Java 18: https://www.oracle.com/java/technologies/downloads/
+    2. Head to the backend folder
+    3. Double click on the jar to turn on the backend
```

Comment:
Maybe say the name of the file that they are supposed to click on? Not sure if they would know what a jar file is